unit/CMakeLists.txt

```diff
@@ -1,10 +1,7 @@
 file(GLOB_RECURSE sources "*.cpp" "*.h")
 
 file(GLOB_RECURSE testing_utils "testing-utils/*.cpp" "testing-utils/*.h")
-
-file(GLOB_RECURSE java_testing_utils
-    "java_bytecode/testing-utils/*.cpp"
-    "java_bytecode/testing-utils/*.h")
+file(GLOB_RECURSE java_tests "java_bytecode/*.cpp" "java_bytecode/*.h")
```

Comment:
I thought it was being removed?

Reply:
This is an annoying quirk of how our CMake files work... It's basically collecting an exclusion list here, so this change expands the exclusion list to include all files in the java_bytecode directory tree (i.e. all the test cases), rather than *just* the java testing utils.

Reply:
Ah ... so a future commit will actually remove those files and these lines as well?